pom.xml

```diff
@@ -637,7 +670,7 @@
 		<tomcat.version>8.5.72</tomcat.version>
 		<mysql.version>8.0.28</mysql.version>
 		<lucene.version>3.6.2</lucene.version>
-		<log4j2.version>2.17.1</log4j2.version>
+		<log4j2.version>2.17.2</log4j2.version>
 		<slf4j.version>1.7.32</slf4j.version>
 		<slack.version>1.18.0</slack.version>
```

Comment:
I think Jackson and Slack have been updated in the dev branch. It might be a good thing to update them here as well.